src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -27,14 +27,15 @@
 public class MainWindow extends UiPart<Stage> {
 
     private static final String FXML = "MainWindow.fxml";
+    private static BrowserPanel browserPanel = new BrowserPanel();
```

Comment:
Hey @zhangriqi, why do we need to make the browserPanel static here? Just wondering!